### ISSUE TITLE
<h2 aria-level=1> should not fail Page needs a level one header

### ISSUE THREAD

**Issue by jnurthen** [dequelabs/axe-core#2348]
Expectation: `<h2 aria-level="1">` should count as a level one header

Actual: results in a "Page must contain a level-one heading" best-practice violation

Motivation: Doesn't seem correct

<pre><code>
axe-core version: 4.5.3
axe-webdriver, extension or other integration version: 3.5.5

Browser and Assistive Technology versions

For Tooling issues:
- Node version: XX  
- Platform:  
</code></pre>

The selector in https://github.com/dequelabs/axe-core/blob/develop/lib/checks/keyboard/page-has-heading-one.json should allow h2:not([role])[aria-level=1] , h3:not([role])[aria-level=1], h4:not([role])[aria-level=1], h5:not([role])[aria-level=1], h6:not([role])[aria-level=1]

**Comment by WilcoFiers** [comment]
Right you are! Thanks for reporting.

**Comment by padmavemulapati** [comment]
verified with the latest axe-core code, it was failing with `<h2 aria-level=1>` 
![image](https://user-images.githubusercontent.com/47409579/87293586-29da0200-c520-11ea-8703-8252ab2d8522.png)

now seeing the fix, its not failing , also showing it into passes:
![image](https://user-images.githubusercontent.com/47409579/87293688-4ece7500-c520-11ea-857d-80b1915f9f90.png)
